### PR TITLE
[Studio] feat: add audit risk insights to the audit dashboard

### PR DESCRIPTION
**Pull request by yx9o**
## What is the purpose of the change

Add an audit risk insights panel to RocketMQ Studio so operators can quickly identify risky audit signals from the current audit filters and visible audit records, including failure rates, partial outcomes, control-plane failures, high-risk operation failures, repeated failed targets, and operator concentration.

## Brief changelog

- Add audit risk insight calculation for filtered audit summaries and current-page audit records.
- Add an Audit Risk Insights panel to the Audit page.
- Add i18n entries for the new audit risk labels and messages.
- Add unit and component tests for risk classification, empty states, and Audit page integration.

## Verifying this change

- `npm test -- src/pages/ops/__tests__/auditRiskInsights.test.ts src/pages/ops/__tests__/AuditRiskInsights.test.tsx src/pages/ops/__tests__/AuditPage.test.tsx`
- `npx eslint src/pages/ops/auditRiskInsightModel.ts src/pages/ops/AuditRiskInsights.tsx src/pages/ops/audit.tsx src/pages/ops/__tests__/auditRiskInsights.test.ts src/pages/ops/__tests__/AuditRiskInsights.test.tsx src/pages/ops/__tests__/AuditPage.test.tsx src/i18n/translations.ts`
- `npm run build`
- `git diff --check apache/rocketmq-studio...HEAD`

### CODE REVIEW

File: web/src/pages/ops/auditRiskInsightModel.ts [hunk 1]
```diff
@@ -0,0 +1,447 @@
+/*
```
Comment (RockteMQ-AI) [inline web/src/pages/ops/auditRiskInsightModel.ts:1]: This module is 447 lines — well-structured with clear separation of types, constants, and logic. If it grows further, consider extracting the threshold constants and type definitions into a dedicated `auditRiskInsightTypes.ts` to keep the model file focused on computation.

File: web/src/pages/ops/auditRiskInsightModel.ts [hunk 2]
```diff
@@ -0,0 +1,447 @@
...
+  | 'HIGH_RISK_FAILURES'
+  | 'REPEATED_TARGET_FAILURES'
+  | 'OPERATOR_CONCENTRATION';
+
+export interface AuditRiskIssue {
+  code: AuditRiskIssueCode;
+  level: AuditRiskLevel;
+  count?: number;
+  percent?: number;
+  operator?: string;
```
Comment (RockteMQ-AI) [inline web/src/pages/ops/auditRiskInsightModel.ts:42]: The `normalizeText` helper is also defined in `dataSourceCoverage.ts` (PR #3129). Consider extracting it to a shared utility (e.g. `utils/text.ts`) to avoid duplication as the codebase grows. Not blocking for this PR.

**Review by RockteMQ-AI — APPROVED** [review]
/tmp/review_work/summary_3131.txt
